tests/app/service/test_service_data_retention_rest.py

```diff
@@ -123,19 +127,25 @@ def test_create_service_data_retention_returns_400_when_notification_type_is_inv
     json_resp = json.loads(response.get_data(as_text=True))
     assert response.status_code == 400
     assert json_resp["errors"][0]["error"] == "ValidationError"
+    type_str = ", ".join(
+        [
+            f"<{type(e).__name__}.{e.name}: {e.value}>"
+            for e in (NotificationType.SMS, NotificationType.EMAIL)
+        ]
+    )
     assert (
         json_resp["errors"][0]["message"]
-        == "notification_type unknown is not one of [sms, email]"
+        == f"notification_type unknown is not one of [{type_str}]"
```

Comment:
Nice, clearer feedback!

Reply:
Unintentional on my part. It was a change that Marshmallow did, which I had to account for.